platform/marvell-armhf/sonic-platform-nokia/7215/scripts/cpu_wdt.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+
+from sonic_platform.chassis import Chassis
+from sonic_py_common import logger
+import time
+import os
+import signal
+import sys
+
+
+TIMEOUT=170
+KEEPALIVE=55
+sonic_logger = logger.Logger('Watchdog')
+sonic_logger.set_min_log_priority_info()
+time.sleep(60)
```

Comment:
@Pavan-Nokia why this sleep needed here?

Reply:
Sonic has "watchdog-control.service" which is designed to disable the watchdog on every boot. Adding this sleep to enable the watchdog after that service has completed.

Using the "After" key word in the service file also does not help as the "after" keyword only assure that our service is started after watchdog-control.service is started and does not ensure that watchdog-control.service is completed before this.